### ISSUE TITLE
[QUESTION] Does "recursive" option remove recursively ?

### ISSUE THREAD

**Issue by nabedge** [bazaarvoice/s3-upload-maven-plugin#11]
Hi.
This is a question.
The "recursive" option will upload files in directories recursively.
And I have an interest in that removed file on source directory will be removed on destination bucket  ?
(like an unix "rsync" command with "--delete" option)

**Comment by matthewbogner** [comment]
No, it will not cause remote files that don't exist locally to be removed from the bucket. 

Docs:
http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/transfer/TransferManager.html#uploadDirectory(java.lang.String, java.lang.String, java.io.File, boolean)

Code:
https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-s3/src/main/java/com/amazonaws/services/s3/transfer/TransferManager.java#L1023
